From 4c9e6d793ff4f13d9137945d113460bc4743ce2d Mon Sep 17 00:00:00 2001
From: Francois Gaffie <francois.gaffie@intel.com>
Date: Mon, 5 Nov 2012 17:47:39 +0100
Subject: [PATCH 07/18] [PORT FROM MAIN] [AUDIO][RENASAS] Enable AudioHAL stub
 for camp

BZ: 66390

This patch avoids to give twice the same lib name.

Change-Id: I0ef3e3482c1042f8afa212f4f3e4028f0625e648
Origin-Change-Id: If2d3240be9640a4b6e826c0315253dffa9c962aa
Signed-off-by: Francois Gaffie <francois.gaffie@intel.com>
Reviewed-on: http://android.intel.com:8080/77382
Reviewed-by: Barthes, FabienX <fabienx.barthes@intel.com>
Tested-by: Barthes, FabienX <fabienx.barthes@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 modules/audio/Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules/audio/Android.mk b/modules/audio/Android.mk
index f773079..8a73756 100644
--- a/modules/audio/Android.mk
+++ b/modules/audio/Android.mk
@@ -22,7 +22,7 @@ LOCAL_PATH := $(call my-dir)
 # required type is 'primary'. Other possibilites are 'a2dp', 'usb', etc.
 include $(CLEAR_VARS)
 
-LOCAL_MODULE := audio.primary.default
+LOCAL_MODULE := audio.primary.stub
 LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
 LOCAL_SRC_FILES := audio_hw.c
 LOCAL_SHARED_LIBRARIES := liblog libcutils
-- 
1.7.9

